core-java-modules/core-java-lang-oop-patterns/src/main/java/com/baeldung/interface_vs_abstract_class/VideoSender.java

```diff
@@ -0,0 +1,13 @@
+package com.baeldung.interface_vs_abstract_class;
+
+import java.io.File;
+
+public class VideoSender implements Sender {
+
+	@Override
+	public void send(File fileToBeSent) {
+		// video sending implementation code
+		System.out.println("Sending Video...");
```

Comment:
Let's also use a proper logger instead of System.out

Reply:
I had added this System.out for testing purpose, now removed it and left the comment alone for simplicity.